cfpb/cfgov-refresh
# Removing handlebars for performance purposes

We really aren't utilizing handlebars so I'm removing it. I'm replacing it with a a lightweight function to replace template tokens.

## Removals

- Removing handlerbars from `package.json`.

## Changes

- Modified `cfgov/unprocessed/js/modules/jquery/cf_notifier.js` to remove `handlebars`. 

## Testing

- Visit `http://localhost:8000/about-us/blog/` and enter `foo@foo`.
- Click the `Sign-up` button.
- Verify that the notification error is being displayed.

## Review

- @contolini 
- @virginiacc 

## Screenshots
<img width="428" alt="screen shot 2016-10-12 at 5 38 19 pm" src="https://cloud.githubusercontent.com/assets/1696212/19328686/d7315194-90a2-11e6-96f3-902f21190b75.png">


## Notes

- Eventually we will be removing `cfgov/unprocessed/js/modules/jquery/cf_notifier.js`

## Checklist
